src/ServiceControl.Transports.ASBS/QueueLengthProvider.cs

```diff
@@ -85,7 +85,7 @@ public Task Start()
                 while (await enumerator.MoveNextAsync().ConfigureAwait(false))
                 {
                     var queueRuntimeProperties = enumerator.Current;
```

Comment:
```suggestion
                    var queueRuntimeProperties = enumerator.Current;
                    if (queuePathToRuntimeInfo.ContainsKey(queueRuntimeProperties.Name))
                    {
                        Logger.InfoFormat("There was an unexpected duplicate queue <{0}> that was retrieved from the Azure Service Bus namespace. The existing properties are going to be updated to the new values.", queueRuntimeProperties.Name);
                        Logger.InfoFormat("{0} queues have currently been retrieved.", queuePathToRuntimeInfo.Count);
                    }
```

Is this something like what you were thinking or is this too wordy?

Reply:
@TravisNickels Yeah, I like it but thinking about it I think we could combine it with a length check and only report is there is a difference and log it as warning:
```suggestion
                    var queueRuntimeProperties = enumerator.Current;
                    if (queuePathToRuntimeInfo.ContainsKey(queueRuntimeProperties.Name))
                    {
                    	var existingItem = queuePathToRuntimeInfo[queueRuntimeProperties.Name];
                    	if(existingItem.Count!=queueRuntimeProperties.Count)
                    	{
                            Logger.WarnFormat("Queue <{0}> already processed with different length (length {1} vs {2})", queueRuntimeProperties.Name, queuePathToRuntimeInfo.Count, existingItem.Count);
                        }
                    }
```